lib/model/TaxonomyPackaging.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:openfoodfacts/interface/JsonObject.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/CountryHelper.dart';
+import 'package:openfoodfacts/utils/TaxonomyQueryConfiguration.dart';
+import 'package:openfoodfacts/utils/TagType.dart';
+
+part 'TaxonomyPackaging.g.dart';
+
+/// Fields of an [TaxonomyPackaging]
+enum TaxonomyPackagingField {
+  ALL,
+  NAME,
+  SYNONYMS,
+  WIKIDATA,
+  CHILDREN,
+  PARENTS,
+  //PACKAGING_MATERIALS, only 4/122 found for roots
+  //NON_RECYCLABLE_AND_NOT_BIODEGRADABLE, only 4/122 found for roots
+  //WEIGHT, only 4/122 found for roots
+  //PACKAGING_SHAPE, only 2/122 found for roots
+}
+
+extension TaxonomyPackagingFieldExtension on TaxonomyPackagingField {
+  static const Map<TaxonomyPackagingField, String> _KEYS =
+      <TaxonomyPackagingField, String>{
+    TaxonomyPackagingField.ALL: 'all',
+    TaxonomyPackagingField.NAME: 'name',
+    TaxonomyPackagingField.SYNONYMS: 'synonyms',
+    TaxonomyPackagingField.WIKIDATA: 'wikidata',
+    TaxonomyPackagingField.CHILDREN: 'children',
+    TaxonomyPackagingField.PARENTS: 'parents',
+    //TaxonomyPackagingField.PACKAGING_MATERIALS: 'packaging_materials',
+    //TaxonomyPackagingField.NON_RECYCLABLE_AND_NOT_BIODEGRADABLE:'non_recyclable_and_non_biodegradable',
+    //TaxonomyPackagingField.WEIGHT: 'weight',
+    //TaxonomyPackagingField.PACKAGING_SHAPE:'packaging_shapes',
```

Comment:
Any reason why they are commented out besides that they are very rare

Reply:
The fact that they are very rare means that they are not reliable.
As I started this PR from json results, I'm just guessing the fields we should use. Maybe there are even other rarer fields.
In first approach, let's not get distracted by useless fields.